docker-compose.yml

```diff
@@ -110,6 +119,11 @@ services:
       action.auto_create_index: "false"
       discovery.type: single-node
       ES_JAVA_OPTS: "-Xms512m -Xmx512m" # less OOM on default settings.
+    healthcheck:
+      test: curl -s http://localhost:9200 >/dev/null || exit 1
```

Comment:
This should be at least `/_cluster/health?wait_for_status=yellow` as elasticsearch can be http accessible before any of the shards are ready.